docs/faq_build.md

```diff
@@ -60,7 +60,12 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="2ff0", MODE:="066
 # tmk keyboard products     https://github.com/tmk/tmk_keyboard
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="feed", MODE:="0666"
 ```
+Or for Infinity Ergodox:
```

Comment:
This should be for all of the Input Club boards, not just the Ergodox Infinity.

Reply:
Adjusted labeling.